app/controllers/claims/support/schools/users_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Claims::Support::Schools::UsersController < Claims::Support::ApplicationController
+  include Claims::BelongsToSchool
+
+  def index
+    @users = @school.users.claims
+  end
+
+  def show
+    @user = Claims::User.find(params.require(:id))
```

Comment:
While we're here. For bit of security, could we do:
```suggestion
    @user = @school.users.find(params.require(:id)).becomes(Placements::User)
```
Then the user will only see Users related to that organisation.

Reply:
Addressed in https://github.com/DFE-Digital/itt-mentor-services/pull/112/commits/38e2682f0db3e76466b875a129b796dd55411c10